### PR TITLE
Updated ModelDeployment Doc

### PR DESCRIPTION
**Pull request by lu-ohai**

### CODE REVIEW

File: docs/source/user_guide/model_deployment/update.rst [hunk 1]
```diff
@@ -1,22 +1,20 @@
...
 Or, you could update the instance shape with:
 
 .. code-block:: python3
 
-    deployment.update(
-        model_deployment_properties.with_instance_configuration(
-            dict(instance_shape="VM.Standard2.1")
-        )
-    )
+    deployment.infrastructure.with_shape_name("VM.Standard.E4.Flex")
```
Comment (mrDzurb) [inline docs/source/user_guide/model_deployment/update.rst:18]: This needs to be checked, if deployment service allow to specify the flex shape without providing memory and ocpus.

**Review by mrDzurb — APPROVED** [review]

File: docs/source/user_guide/model_deployment/update.rst [hunk 2]
```diff
@@ -1,22 +1,20 @@
 Update
 ******
 
-The ``.update()`` method of the ``ModelDeployment`` class is used to make changes to a deployed model. This method accepts the same parameters as the ``.deploy()`` method. Check out the `Editing Model Deployments <https://docs.oracle.com/en-us/iaas/data-science/using/model_dep_manage.htm>`__ for a
+The ``.update()`` method of the ``ModelDeployment`` class is used to make changes to a deployed model. Check out the `Editing Model Deployments <https://docs.oracle.com/en-us/iaas/data-science/using/model_dep_manage.htm>`__ for a
 list of what properties can be updated.
 
-A common use case is to change the underlying model that is deployed. In the following code snippets, the variable ``deployment`` is a ``ModelDeployment`` object.  This object can be obtained from a call to ``.deploy()`` or ``.get_model_deployment()``.
+A common use case is to change the underlying model that is deployed. In the following code snippets, the variable ``deployment`` is a ``ModelDeployment`` object.  This object can be obtained from a call to ``.deploy()`` or ``.from_id()``.
```
Comment (jdesanto) [inline docs/source/user_guide/model_deployment/update.rst:7]: This should make explicit reference to the code that follows where a change is actually made.

**Review by jdesanto — APPROVED** [review]

File: docs/source/user_guide/model_registration/model_deploy_byoc.rst [hunk 3]
```diff
@@ -0,0 +1,187 @@
+Deploy Model on Container Runtime
+*********************************
+
+The ADS ``GenericModel`` and ``ModelDeployment`` classes allow you to run a container image using OCI data science model deployment.
```
Comment (mayoor) [inline docs/source/user_guide/model_registration/model_deploy_byoc.rst:4]: allow you to deploy model using container

File: docs/source/user_guide/model_registration/model_deploy.rst [hunk 4]
```diff
@@ -1,3 +1,6 @@
+Deploy Model on Conda Runtime
```
[the remark this answers is not in the document]
Comment (lu-ohai) [reply docs/source/user_guide/model_registration/model_deploy.rst]: Updated

File: docs/source/user_guide/model_registration/model_deploy_byoc.rst [hunk 5]
```diff
@@ -0,0 +1,187 @@
+Deploy Model on Container Runtime
```
[the remark this answers is not in the document]
Comment (lu-ohai) [reply docs/source/user_guide/model_registration/model_deploy_byoc.rst]: Updated

File: docs/source/user_guide/model_registration/model_deploy_byoc.rst (hunk 3 above)
Comment (lu-ohai) [reply docs/source/user_guide/model_registration/model_deploy_byoc.rst]: Updated

**Review by mayoor — APPROVED** [review]

**Pull request merged**